### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by vaga3461**
Pull request for week1 problem set; needs peer reviews

### CODE REVIEW

**Comment by vaga3461** [comment]
Binder link for reviewers: <a href='https://mybinder.org/v2/gh/CUB-Computational-Tools/assignment_problem_set/binder-python?urlpath=git-pull?repo=https%3A%2F%2Fgithub.com%2FCUB-Computational-Tools%2Fassignment_problem_set%26branch%3Dweek1%26urlpath%3Dlab'></a>

File: week1.ipynb [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    "\n",
+    "where $\\textit{m}$ is a mass with units kg, and $\\textit{v}$ is a velocity with units m/s. By simplifying the units in this equation, we can see that kinetic energy has units $kg\\cdot m/s^2$. We can also easily see that when an object is not moving ($v = 0$), its kinetic energy is zero.\n",
+    "\n",
+    "### Work\n",
+    "\n",
+    "Work is a force acting over a distance. Specifically, it is calculated from the displacement of an object parallel to the force acting on it. In this lesson we will only condsider forces acting in the vertical plane. Work is most generally written as\n",
+    "\n",
+    "$W = F*d$\n",
+    "\n",
+    "where $F$ is a force and $d$ is the displacement, or the distance over which the force acts. We can see from this equation that work is zero if the displacement of an object is zero, $\\textit{whether or not a force has been exerted on that object}$. This means that for all the energy you would exert trying to push down a wall in your house, you aren't doing any work! \n",
```
Comment (KevRozmiarek) [inline week1.ipynb]: The textit looks awkward in the complied doc

File: week1.ipynb [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    "\n",
+    "Kinetic energy is the energy that an object possesses due to motion. Kinetic energy is given by the equation\n",
+    "\n",
+    "$KE = \\frac{1}{2}mv^2$\n",
+    "\n",
+    "where $\\textit{m}$ is a mass with units kg, and $\\textit{v}$ is a velocity with units m/s. By simplifying the units in this equation, we can see that kinetic energy has units $kg\\cdot m/s^2$. We can also easily see that when an object is not moving ($v = 0$), its kinetic energy is zero.\n",
+    "\n",
+    "### Work\n",
+    "\n",
+    "Work is a force acting over a distance. Specifically, it is calculated from the displacement of an object parallel to the force acting on it. In this lesson we will only condsider forces acting in the vertical plane. Work is most generally written as\n",
```
Comment (KevRozmiarek) [inline week1.ipynb]: It is just displacement. The word parallel is confusing here.

File: week1.ipynb [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    "Why is work negative here? This is a sign convention used because an object that has fallen has $\\textit{lost}$ potential energy, so $\\Delta PE$ is negative. Adding the minus sign in front of $W$ ensures that in such a case, the minus signs cancel, and the work done is still postive."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## An example\n",
+    "\n",
+    "Let's imagine an apple falling to the ground from a height of 10 feet. Its mass is 100 g. We will calculate how much work was done on the apple, and in the question below you will find the change in velocity that it experienced during its fall. We will assume the apple was staionary immediately before falling."
```
Comment (KevRozmiarek) [inline week1.ipynb]: "stationary"

File: week1.ipynb [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Questions\n",
+    "\n",
+    "1. Please describe the types of energy held in the following scenarios. Do you see anything problematic in any of them, given only the information we've learned today?\n",
+    "- A box sitting stationary on a table\n",
```
Comment (KevRozmiarek) [inline week1.ipynb]: I think you might have to explain a reference frame more explicitly to ask this question. I imply it in the next one though!

**Review by KevRozmiarek — CHANGES_REQUESTED** [review]
Overall, great work! I feel like I should personally provide more background on mine having read this.....

**Review by splazatorres — COMMENTED** [review]
Hi. For some reason, the comments (I made) on the ReviewNB page are not showing up here. let me know if you can access them or if you don't see them. Also, great work! I really loved how well explained, clear, and well-thought-out the notebook was!

**Comment by vaga3461** [comment]
Thanks, Stephanie! Yes, I was able to see your comments on ReviewNB. Not sure why they showed up in different places, but I found them all :)

**Comment by splazatorres** [comment]
Awesome. Also, I couldn't access your notebook in binder. Seb let me know it might be because you still have the original template name in the README.md file (the assignment-problem-set instead of your repo name).

**Comment by vaga3461** [comment]
Reviewers: please use this link for binder! https://mybinder.org/v2/gh/CUB-Computational-Tools/2020-ps-vaga3461/binder-python?urlpath=git-pull?repo=https%3A%2F%2Fgithub.com%2FCUB-Computational-Tools%2F2020-ps-vaga3461%26branch%3Dweek1%26urlpath%3Dlab

**Comment by vaga3461** [comment]
About to merge - wish me luck

**Pull request merged**